openci-runner/apps/lifeguard/readme.md

```
@@ -0,0 +1,25 @@
+
+> This readme is automatically generated by [create-ink-app](https://github.com/vadimdemedes/create-ink-app)
+
+## Install
+
+```bash
+$ npm install --global lifeguard
+```
+
+## CLI
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)